app/app.rb

```diff
@@ -0,0 +1,55 @@
+require 'sinatra/base'
+require './app/data_mapper_setup.rb'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  set    :session_secret, 'super secret'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.